src/components/Logout.vue

```diff
@@ -0,0 +1,13 @@
+<script setup lang="ts">
+import { useMainStore } from '@/stores/useMainStore';
+
+const store = useMainStore()
+const { logoutCurrentUser } = store
+
+</script>
+
+<template>
+	<button class="opacity-50" @click="logoutCurrentUser()">
```

Comment:
It's better to integrate it in the component you created for the dropdown, creating a component just for this button seems too much :/